index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<head>
+    <title>
+        freeCodeCamp Survey Form
+    </title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">
+        Thank you for
+        taking the time to help us
+        improve the platform
+    </p>
+    <form id="survey-form">
+        <label for="name" id = "name-label">Name</label>
+        <input id="name" placeholder= "Enter your name" type="text" required>
+
+        <label for="email" id = "email-label">email</label>
+        <input id="email" placeholder= "Enter your email" type="email" required>
+
+        <label for="number" id = "number-label">age</label>
+        <input id="number" placeholder= "age" type="number" min ="10" max ="99">
+
+        <select name="dropdown" id="dropdown">
+          <option value="Challenges">Challenges</option>
+          <option value="Challenges">Projects</option>
+          <option value="Community">Community</option>
+          <option value="Open Source">Open Source</option>
+        </select>
+      <fieldset>
+        <legend>choose gender</legend>
+        <label for = 'Male'> Male</label>
+        <input type="radio" name="Group" value="male">
+          <label for = 'Female'> Female</label>
+        <input type="radio" name="Group" value="female">
+          <label for = 'Other'>Other</label>
+        <input type="radio" name="Group" value="other">
+      </fieldset>
+
+
+  <label for = 'choose from here'>Choose from here:</label> <br>
+<label><input type="checkbox" value="first_checkbox"> checkbox1</label><br>
+<label><input type="checkbox" value="second_checkbox"> checkbox2</label><br>
+<label><input type="checkbox" value="Third_checkbox"> checkbox3</label><br>
+
+  <label for = 'choose from here'>Write some comment here !</label> <br>
+    <textarea name="write the description" rows="8" cols="80"></textarea>
+
+<button type="input" id="submit" name="button">Submit</button>
+    </form>
+</body>
```

Comment:
It might be a good idea to always close the opened tags. Kindly add a `</html>` tag in the last line. This will help on avoiding errors and make your code cleaner! :star_struck: